test/Core.Test/Utilities/StaticStoreTests.cs

```diff
@@ -51,7 +51,7 @@ public void StaticStore_GlobalEquivalentDomains_OnlyAsciiAllowed()
             for (var i = 0; i < domain.Length; i++)
             {
                 var character = domain[i];
-                Assert.True(char.IsAscii(character), $"Domain: {domain} contains non-ascii character at index {i}");
+                Assert.True(char.IsAscii(character), $"Domain: {domain} contains non-ASCII character: '{character}' at index {i}");
```

Comment:
```suggestion
                Assert.True(char.IsAscii(character), $"Domain: {domain} contains non-ASCII character: '{character}' at index: {i}");
```
⛏️ Is this the norm?

Reply:
The norm would be to not generally add your own message and let XUnit make one for you but in this case the code I would normally write (`Assert.All(domain, c => Assert.True(char.IsAscii(c));`) would result in this error message:
```
Error Message:
   Assert.All() Failure: 1 out of 10 items in the collection did not pass.
[5]: Item:  'ց'
     Error: Assert.True() Failure
            Expected: True
            Actual:   False
```
so that is what I would call the norm but I think showing the entire domain somewhere is useful so I think we can kinda do whatever we want.